dateparser/search/search.py

```diff
@@ -86,10 +86,12 @@ def split_if_not_parsed(self, item, original):
                 possible_splits.extend(self.split_by(item, original, splitter))
         return possible_splits
 
-    def parse_item(self, parser, item, translated_item, parsed, need_relative_base):
+    def parse_item(self, parser, item, translated_item, parsed, need_relative_base, language):
```

Comment:
```suggestion
    def parse_item(self, parser, item, translated_item, parsed, need_relative_base, language=None):
```

Again, this is to avoid a breaking change (as this function is declared as public, even if it should be private).